remote_client.md

```diff
@@ -0,0 +1,69 @@
+# Podman remote-client tutorial
+
+### What is the remote-client
+
+First and foremost, the remote-client is under heavy development.  We are adding new
+commands and functions frequently.  We also are working on a rootless implementation that
+does not require privileged users.
+
+The purpose of the Podman remote-client is to allow users to interact with a Podman "backend"
+while one a separate client.  The command line interface of the remote client is exactly the
+same as the regular Podman for commands that are implemented.
+
```

Comment:
Suggest changing the last line to "The command line interface of the remote client's commands are exactly the same as the corresponding commands for Podman."  That way you won't have to touch it up later when the commands are all implemented, or if you find one or two that can't be implemented.